src/Fl_Text_Editor.cxx

```diff
@@ -132,7 +132,9 @@ static struct {
   { FL_Page_Down, FL_CTRL|FL_SHIFT,         Fl_Text_Editor::kf_c_s_move   },
 //{ FL_Clear,     0,                        Fl_Text_Editor::delete_to_eol },
   { 'z',          FL_CTRL,                  Fl_Text_Editor::kf_undo       },
-  { '/',          FL_CTRL,                  Fl_Text_Editor::kf_undo       },
+  { 'z',          FL_CTRL|FL_SHIFT,         Fl_Text_Editor::kf_redo       }, // Windows users may be used to Ctr-Y
```

Comment:
s/Ctr-Y/Ctrl-Y/

Hmm, this comment is true, but what are the consequences? Do we ignore this fact?

Reply:
I added this specifically for MS Windows (Screen Driver)